core/templates/dev/head/components/rubrics-editor/rubrics-editor.directive.html

```diff
@@ -11,11 +11,11 @@
              title="Edit Rubric Explanation">&#xE254;
           </i>
         </div>
-        <span ng-if="$ctrl.isExplanationEmpty($ctrl.getRubrics()[$index].getExplanation()) && $index === 1"
+        <span ng-if="$ctrl.isExplanationEmpty($ctrl.getRubrics()[$index].getExplanation()) && $index === 1 && !$ctrl.inSkillEditor"
```

Comment:
This seems weird. I think you want the framing to be something that's more like "newSkillIsBeingCreated" -- right?

Ditto below. Or are there other places besides the skill editor and the new creation flow where this is shown?

Reply:
No, there aren't (at least, not yet). If it is in the skill editor, then it is sure that the skill is already created in which case, the other instruction is not applicable, but anywhere else, if this is shown, then it can only be part of creating a new skill, hence the name.
Currently, it is only shown in the topics and skills dashboard, when creating a new skill and in the skill editor, when editing it. Should I rename it?

Reply:
Yes, please rename. I think the thing you want to call out specifically is "the new skill creation process" vs "everything else".

Reply:
Done